kong/dao/schemas/upstreams.lua

```diff
@@ -84,9 +84,9 @@ local healthchecks_defaults = {
     },
     unhealthy = {
       http_statuses = { 429, 500, 503 },
-      tcp_failures = 2,
-      timeouts = 7,
-      http_failures = 5,
+      tcp_failures = 0,  -- 0 = circuit-breaker disabled by default
+      timeouts = 0,      -- 0 = circuit-breaker disabled by default
+      http_failures = 0, -- 0 = circuit-breaker disabled by default
```

Comment:
Since I heard you mentioning that setting those values to `0` was also a demonstration of us being unopinonated about those values (so that users are not influenced by our defaults), should we not also update `active.unhealthy.{tcp/http}_failures` to `0`?

Also, regarding the default successes: `active.healthy.successes` is `2` and `passive.health.successes` is `5`. Should they be the same? Should they be `0` as well for unopinionated defaults?

Reply:
> Since I heard you mentioning that setting those values to 0 was also a demonstration of us being unopinonated about those values (so that users are not influenced by our defaults), should we not also update active.unhealthy.{tcp/http}_failures to 0?

Well, that was a nice side-effect, but not the primary goal. :) Do you think we should pursue that direction?

> Should they be 0 as well for unopinionated defaults?

They can be. But then you have to remember to set both `success` and `failure` values when enabling health checks or else you'll be able to bring nodes down but not back up.

Reply:
> Do you think we should pursue that direction?

Not necessarily, I just wanted to get your feedback on the idea!

Reply:
I think we should keep providing a default set of HTTP status codes, since it's easy to miss some. For this initial round of the functionality, all defaults as 0 may be the safest bet as we learn in the field what works best. 

We could change that in the future (e.g. if we get around to add shortcut `enabled` config fields for usability, which you suggested once, then we could provide default values for the disabled-by-default counters as a suggestion to users.)

Reply:
Yeah, SGTM, I'll go ahead and approve this because none of these are blockers to us anyway, was we were mostly concerned about `0`s for the unhealthyness. I''ll defer to you if you want everything disabled by default or not!